tests/test_classifier/test_class_prediction_error.py

```diff
@@ -0,0 +1,98 @@
+# tests.test_classifier.test_rocauc
+# Testing for the ROCAUC visualizer
+#
+# Author:   Benjamin Bengfort <bbengfort@districtdatalabs.com>
+# Author:   Rebecca Bilbro <rbilbro@districtdatalabs.com>
+# Created:  Tue May 23 13:41:55 2017 -0700
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: test_rocauc.py [] benjamin@bengfort.com $
+
+"""                                                                                                                  Testing for the ClassPredictionError visualizer
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import pytest
+import numpy as np
+import numpy.testing as npt
+
+from tests.base import VisualTestCase
+from tests.dataset import DatasetMixin
+from yellowbrick.classifier.rocauc import *
+from yellowbrick.exceptions import ModelError
+
+from sklearn.svm import LinearSVC
+from sklearn.naive_bayes import MultinomialNB
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.datasets import load_breast_cancer
+from sklearn.linear_model import LogisticRegression
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.model_selection import train_test_split as tts
+from yellowbrick.classifier.class_balance import *
+from tests.base import VisualTestCase
+from sklearn.svm import LinearSVC
+
+##########################################################################
+## Data
+##########################################################################
+
+X = np.array(
+        [[ 2.318, 2.727, 4.260, 7.212, 4.792],
+         [ 2.315, 2.726, 4.295, 7.140, 4.783,],
+         [ 2.315, 2.724, 4.260, 7.135, 4.779,],
+         [ 2.110, 3.609, 4.330, 7.985, 5.595,],
+         [ 2.110, 3.626, 4.330, 8.203, 5.621,],
+         [ 2.110, 3.620, 4.470, 8.210, 5.612,]]
+    )
+
+y = np.array([1, 1, 0, 1, 0, 0])
+
+##########################################################################
+##  Tests
+##########################################################################
+
+class ClassPredictionErrorTests(VisualTestCase):
+
+    @pytest.mark.skip(reason="not implemented yet")
+    def test_class_report(self):
```

Comment:
`test_class_prediction_error`?

Reply:
not quite sure what you are asking here... 😕